app/models/alchemy/dragonfly_signature.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module Alchemy
+  class DragonflySignature < ActiveRecord::Base
```

Comment:
Could we have ActiveModel validations for the presence and uniqueness constraints enforced by the schema?

Reply:
initially I had not included these on purpose to keep this model as light as possible.
I've updated it to at least cascade delete the DragonflySignatures when image versions are deleted.